src/modules/toast/handler/handlerToast.js

```diff
@@ -6,6 +6,14 @@ const ERROR = 'error';
 const WARNING = 'warning';
 const INFO = 'info';
 const SENTRY = 'sentry';
+const GLOBAL_ERRORS = {
+  "Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced.":
+    "Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced."
```

Comment:
did you mean to write the same error twice?

Reply:
Yeah, it was done intentionally because it's a JS object so we can quickly check object.values.  Same pattern as in some of the other errorHandlers.